Assets/UI/worldinput_CQUI.lua

```diff
@@ -0,0 +1,559 @@
+        -- Use false as we need to let the unmodifed script deal with the Alt Key
+        cquiHandledKey = false;
+      elseif (CQUI_cityview) then
+        LuaEvents.CQUI_GoPrevCity();
+        cquiHandledKey = true;
+      else
+        UI.SelectPrevReadyUnit();
+        cquiHandledKey = true;
+      end
+    end
+
+    if (uiKey == Keys.VK_SHIFT and ContextPtr:LookUpControl("/InGame/TechTree"):IsHidden() and ContextPtr:LookUpControl("/InGame/CivicsTree"):IsHidden()) then
+      cquiHandledKey = true;
+      if (CQUI_cityview) then
+        UI.SetInterfaceMode(InterfaceModeTypes.SELECTION);
+        UI.SelectNextReadyUnit();
+      else
+        LuaEvents.CQUI_GoNextCity();
+      end
+    end
+
+    if (keyPanChanged == true) then
+      -- Base game file uses m_edgePanX and m_edgePanY... but I do not see evidence where these are assigned to, except for the initial 0.
+      ProcessPan(0, 0);
+      cquiHandledKey = true;
+    end
+  else -- CQUI_hotkeyMode == CQUI_HOTKEYMODE_CLASSIC  (Classic binds that overlap with enhanced binds)
+    if (uiKey == Keys.Q) then
+      if (unitType == "UNIT_BUILDER") then
+        CQUI_BuildImprovement(UI.GetHeadSelectedUnit(), GameInfo.Improvements["IMPROVEMENT_QUARRY"].Hash);
+        cquiHandledKey = true;
+      end
+    end
+
+    if (uiKey == Keys.S) then
+      if (formationClass == "FORMATION_CLASS_AIR") then
+        -- M4A: Not certain that this command actually is functional
+        UnitManager.RequestCommand(UI.GetHeadSelectedUnit(), UnitOperationTypes.AIR_ATTACK);
+        cquiHandledKey = true;
+      end
+    end
+  end -- Classic binds that overlap with enhanced binds
+
+    -- Focus Capital hotkey
+  if (uiKey == Keys.VK_HOME) then
+    UI.SelectCity(Players[Game.GetLocalPlayer()]:GetCities():GetCapitalCity());
+    cquiHandledKey = true;
+  end
+
+  if (uiKey == Keys.VK_BACK) then
+    if (unitType ~= nil) then
+      UnitManager.RequestCommand(UI.GetHeadSelectedUnit(), UnitCommandTypes.CANCEL);
+      cquiHandledKey = true;
+    end
+  end
+
+  if (uiKey == Keys.C and CQUI_isAltDown == true) then
+    if (unitType == "UNIT_BUILDER") or (unitType == "UNIT_MILITARY_ENGINEER") then
+      UnitManager.RequestOperation(UI.GetHeadSelectedUnit(), UnitOperationTypes.HARVEST_RESOURCE);
+      UnitManager.RequestOperation(UI.GetHeadSelectedUnit(), UnitOperationTypes.REMOVE_FEATURE);
+      -- Use false as we need to let the unmodifed script deal with the Alt Key
+      cquiHandledKey = false;
+    end
+  end
+
+  if (uiKey == Keys.F) then
+    if (unitType == "UNIT_BUILDER") then
+      CQUI_BuildImprovement(UI.GetHeadSelectedUnit(), GameInfo.Improvements["IMPROVEMENT_FISHING_BOATS"].Hash);
+      cquiHandledKey = true;
+    elseif (unitType == "UNIT_MILITARY_ENGINEER") then
+      CQUI_BuildImprovement(UI.GetHeadSelectedUnit(), GameInfo.Improvements["IMPROVEMENT_FORT"].Hash);
+      cquiHandledKey = true;
+    end
+  end
+
+  if (uiKey == Keys.H and unitType == "UNIT_BUILDER") then
```

Comment:
I think original had the H key assigned to fortify until healed for damaged military/religious units. Was this one of the ones you intentionally removed?
```
  if( uiKey == Keys.H ) then
    if (unitType ~= nil) then -- OPTIMIZATION -- if unit health 100% alert not heal
      UnitManager.RequestOperation(UI.GetHeadSelectedUnit(), 2126026491); -- OH MY GOD IT WAS SO HARD TO FIND THAT ID. WHAT WAS FIRAXIS THINKING
    end
    return;
  end
```

Reply:
Yep - the comment there was funny though.  The game handles the H key in the same manner, so no reason to handle it here.